<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSongProperties</class>
 <widget class="QDialog" name="DlgSongProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Song properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hash:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Title [Parsed ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Author [Parsed ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Author [manual]:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Title [manual]:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="leId3Genre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Genre, as detected from the entire ID3 tag</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Author [ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1" colspan="3">
      <widget class="QLineEdit" name="leFilenameAuthor">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>MPM [ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="3">
      <widget class="QLineEdit" name="leId3Title"/>
     </item>
     <item row="20" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>MPM [filename]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>MPM [manual]:</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Genre [filename]:</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1" colspan="3">
      <widget class="QLineEdit" name="leFilenameTitle">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="20" column="3">
      <widget class="QLineEdit" name="leFilenameMeasuresPerMinute">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Title [filename]:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QLineEdit" name="leId3MeasuresPerMinute"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Genre [manual]:</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Author [filename]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cbManualGenre"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Genre [ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLineEdit" name="leFilenameGenre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Title [ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="3">
      <widget class="QLineEdit" name="leId3Comment"/>
     </item>
     <item row="11" column="1" colspan="3">
      <widget class="QLineEdit" name="leId3Author"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Comment [ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Genre [Parsed ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1" colspan="3">
      <widget class="QLineEdit" name="lePid3Title">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="lePid3Genre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>MPM [Parsed ID3]:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1" colspan="3">
      <widget class="QLineEdit" name="lePid3Author">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="17" column="3">
      <widget class="QLineEdit" name="lePid3MeasuresPerMinute">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="4" rowspan="4">
      <widget class="QPushButton" name="btnCopyId3Tag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="5">
      <widget class="QListWidget" name="lwDuplicates">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="4">
      <widget class="QPlainTextEdit" name="pteNotes"/>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QLineEdit" name="leManualMeasuresPerMinute"/>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QLineEdit" name="leManualTitle"/>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="leManualAuthor"/>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="leLength">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="leHash">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shared Data:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="5">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Duplicates:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="4" rowspan="3">
      <widget class="QPushButton" name="btnCopyPid3Tag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="18" column="4" rowspan="3">
      <widget class="QPushButton" name="btnCopyFilenameTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actRemoveFromLibrary">
   <property name="text">
    <string>Remove from library...</string>
   </property>
   <property name="toolTip">
    <string>Removes the song from the library, keeps the disk file</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actDeleteFromDisk">
   <property name="text">
    <string>Delete from disk...</string>
   </property>
   <property name="toolTip">
    <string>Removes song from library AND deletes the file from the disk</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>leHash</tabstop>
  <tabstop>leLength</tabstop>
  <tabstop>leManualAuthor</tabstop>
  <tabstop>leManualTitle</tabstop>
  <tabstop>cbManualGenre</tabstop>
  <tabstop>leManualMeasuresPerMinute</tabstop>
  <tabstop>pteNotes</tabstop>
  <tabstop>lwDuplicates</tabstop>
  <tabstop>leId3Author</tabstop>
  <tabstop>leId3Title</tabstop>
  <tabstop>leId3Comment</tabstop>
  <tabstop>leId3Genre</tabstop>
  <tabstop>leId3MeasuresPerMinute</tabstop>
  <tabstop>lePid3Author</tabstop>
  <tabstop>lePid3Title</tabstop>
  <tabstop>lePid3Genre</tabstop>
  <tabstop>lePid3MeasuresPerMinute</tabstop>
  <tabstop>leFilenameAuthor</tabstop>
  <tabstop>leFilenameTitle</tabstop>
  <tabstop>leFilenameGenre</tabstop>
  <tabstop>leFilenameMeasuresPerMinute</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
